clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -282,7 +282,7 @@ static KafkaAdminClient createInternal(AdminClientConfig config) {
 
         try {
             metadata = new Metadata(config.getLong(AdminClientConfig.RETRY_BACKOFF_MS_CONFIG),
-                    config.getLong(AdminClientConfig.METADATA_MAX_AGE_CONFIG), false);
+                    config.getLong(AdminClientConfig.METADATA_MAX_AGE_CONFIG), true);
```

Comment:
Can we please add a comment here stating that even though we allow auto creation, we never set `topics`, so we never trigger it.